release-notes.txt

```diff
@@ -0,0 +1,169 @@
+BISDN Linux
+4.7.0 Release Notes
+Date: 2022-08-03
+
+! New in this release
+- Yocto: Updated Yocto from 3.1 (dunfell) to 4.0 (kirkstone).
+  This includes major version updates for most OS components.
+- Linux kernel: Updated to 5.15.56.
+- FRR: Updated to 8.3.
+
+! Changed behavior
+- MAC addresses of port* interfaces are now set based on EEPROM contents instead
```

Comment:
```suggestion
- MAC addresses of Port* interfaces are now set based on EEPROM contents instead
```

Reply:
The interfaces use lower case names, so I think this should stay lower case.